src/api/config/routes.rb

```diff
@@ -462,6 +460,8 @@ def self.public_or_about_path?(request)
   get 'published/:project(/:repository(/:arch(/:binary)))' => 'published#index', constraints: cons
   get 'published/' => 'source#index', via: :get
 
+  resources :staging_workflows, except: [:index], controller: 'webui/staging_workflows', constraints: cons
```

Comment:
You also don't have an ``:edit``, right?

Reply:
No